stdlib/public/runtime/Errors.cpp

```diff
@@ -98,7 +100,7 @@ static bool getSymbolNameAddr(llvm::StringRef libraryName,
                                 sizeof(szUndName), dwFlags);
   });
 
-  if (dwResult == TRUE) {
```

Comment:
`dwResult` here is a `DWORD`, not a `BOOL`, and its value on success is a non-zero length, not `TRUE`. So this `if` statement was broken.

Reply:
Bleh, so it was.  Thanks for fixing this!